lib/tasks/gias_update.rake

```diff
@@ -1,14 +1,10 @@
 desc "Upsert GIAS data into GiasSchool"
 task gias_update: :environment do
-  today = Time.zone.today.strftime("%Y%m%d")
-  gias_filename = "edubasealldata#{today}.csv"
+  tempfile = Gias::CsvDownloader.call
+  Rails.logger.info "GIAS CSV Downloaded!"
 
-  Rails.logger.info "Downloading the new gias file for #{Time.zone.today}"
-  tempfile = Down.download("#{ENV["GIAS_CSV_BASE_URL"]}/#{gias_filename}")
-  Rails.logger.info "Done!"
-
-  Rails.logger.info "Importing data"
-  GiasCsvImporter.call(tempfile.path)
+  Rails.logger.info "Importing GIAS data"
+  Gias::Importer.call(tempfile.path)
 
   tempfile.unlink
 end
```

Comment:
Do we actually still need this Rake task? 🤔

If we think it's still valuable to keep, I guess it could actually be simplified to:

```ruby
task gias_update: :environment do
  Gias::SyncAllSchoolsJob.perform_now
end
```